model/src/main/kotlin/OrtResult.kt

```diff
@@ -372,7 +372,7 @@ data class OrtResult(
      */
     @JsonIgnore
     fun getProjects(omitExcluded: Boolean = false): Set<Project> =
-        analyzer?.result?.withScopesResolved()?.projects.orEmpty().filterTo(mutableSetOf()) { project ->
+        analyzer?.result?.projects.orEmpty().filterTo(mutableSetOf()) { project ->
```

Comment:
Commit message: It sounds like it was impossible to access projects if the ORT result is in dependency graph format, but I believe what you mean is that it is impossible to access projects without converting them to the scope format.

Reply:
Reworded the commit message to make the intension (hopefully) clearer.